### DISCUSSION TITLE
Setting breakpoints in Grid

### DISCUSSION THREAD

**Discussion by somguyth** [chakra-ui/chakra-ui#4389] [Q&A] [answered]
Is there a way I can set breakpoints in `Grid` so that it automatically resizes? I've been able to get it to work with the `Box` example by following the docs, but I couldn't see any analogue in the docs pertaining to `Grid`.

**Comment by aaronkik** [comment, marked as answer]
You can do something like this and apply it to other props of the Grid component.

```javascript
<Grid templateColumns={{base: "repeat(1, 1fr)", md:"repeat(5, 1fr)"}} gap={6}>
  <Box w="100%" h="10" bg="blue.500" />
  <Box w="100%" h="10" bg="blue.500" />
  <Box w="100%" h="10" bg="blue.500" />
  <Box w="100%" h="10" bg="blue.500" />
  <Box w="100%" h="10" bg="blue.500" />
</Grid>
```
